### PR TITLE
Added blobstore image example

### PR DESCRIPTION
**Pull request by thesandlord**

### CODE REVIEW

File: appengine/images/api/blobstore.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+"""
+Sample application that demonstrates how to use the App Engine Images API.
+
+For more information, see README.md.
+"""
+
+# [START all]
+# [START thumbnailer]
+from google.appengine.api import images
+from google.appengine.ext import blobstore
```
Comment (jonparrott) [inline appengine/images/api/blobstore.py]: blobstore should be above images, lint will fail otherwise.

**Comment by jonparrott** [comment]
LGTM with small nit

File: appengine/images/api/blobstore_test.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+@pytest.fixture
+def app(testbed):
+    return webtest.TestApp(blobstore.app)
+
+
+def test_img(app):
+    with mock.patch('blobstore.images') as mock_images:
+        with mock.patch('blobstore.blobstore') as mock_blobstore:
```
Comment (waprin) [inline appengine/images/api/blobstore_test.py]: can we make these decorators? less nesting so a little cleaner imo

File: appengine/images/api/blobstore.py (hunk 1 above)
Comment (thesandlord) [inline appengine/images/api/blobstore.py]: Lint passes for me?

Comment (jonparrott) [inline appengine/images/api/blobstore.py]: Sorry, I'm dumb and mis-read ext and api. This is fine.

File: appengine/images/api/blobstore_test.py (hunk 2 above)
Comment (thesandlord) [inline appengine/images/api/blobstore_test.py]: Can do this after closing the bug

**Comment by thesandlord** [comment]
Can we merge this?

**Comment by jonparrott** [comment]
Sure.

**Pull request merged**